setup.cfg

```diff
@@ -12,6 +12,7 @@ classifiers =
     Programming Language :: Python :: 3.8
```

Comment:
I don't seem to be able to comment on the right line, so commenting here: Should you remove 3.6?

Reply:
Yeah good point I did wonder about that and decided to leave it in because we still technically support it. But thinking about it again, I guess it would make sense to remove given that these metadata are probably mostly used to filter packages on PyPI and we don't ship wheels anymore for that version (although you would still be able to compile with the source distrib).

Reply:
Actually, it's a good point. But OCIO doesn't set the https://packaging.python.org/en/latest/specifications/core-metadata/#requires-python metadata, which is kind of dangerous to not set. The classifiers are just used for actually filtering stuff. `requires-python` tells installers which Python version the package supports.

Reply:
I guess this was what the .toml [project] section was defining before, is that [python-requires](https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#python-requires)? Looks like fairly new addition.

Reply:
Oh yeah indeed. I don't know what I was thinking about. Anyway, it should be moved into `setup.cfg` for it to be included in the metadata, as per the warning I pointed to.